Makefile

```diff
@@ -190,7 +190,15 @@ docker:
 	curl https://get.docker.com/builds/Linux/x86_64/docker-1.9.1 -o docker
 	chmod +x docker
 
-semaphore:
+docker-1-10:
```

Comment:
Same comments as calico-containers

Did you look at adding an ST to test the `--ip` option on docker run?